src/Database/Model.php

```diff
@@ -320,4 +320,19 @@ protected function throwErrorMessages() : void
             getShortClassName(new static) . ' ' . current($this->getMessages())->getMessage()
         );
     }
+
+    /**
+     * hasProperty.
+     *
+     * @param  string $property
+     *
+     * @return bool
+     */
+    public function hasProperty(string $property) : bool
+    {
+        $model = self::findFirstOrFail();
+        $metadata = $model->getModelsMetaData();
+        $attributes = $metadata->getAttributes($model);
```

Comment:
@FredPeal why cant you pass $this instead of $model? this is not a static method therefore you need a object to use